internal/pkg/action/operator_uninstall.go

```diff
@@ -5,33 +5,29 @@ import (
 	"fmt"
 	"strings"
 
-	v1 "github.com/operator-framework/api/pkg/operators/v1"
 	"github.com/operator-framework/api/pkg/operators/v1alpha1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/operator-framework/kubectl-operator/internal/pkg/operand"
 	"github.com/operator-framework/kubectl-operator/pkg/action"
 )
 
 type OperatorUninstall struct {
-	config *action.Configuration
-
-	Package                  string
-	DeleteAll                bool
-	DeleteCRDs               bool
-	DeleteOperatorGroups     bool
-	DeleteOperatorGroupNames []string
+	Package         string
+	config          *action.Configuration
+	operandLister   *action.OperatorListOperands
+	OperandStrategy operand.DeletionStrategy
```

Comment:
Nit: keep the unexported fields separate from the exported fields, and try to keep both sets of fields in an API-user-friendly ordering (e.g. in decreasing order of how often we expect the field to be set or used)